css/properties/background-image.json

```diff
@@ -135,7 +135,8 @@
               "edge": "mirror",
               "firefox": {
                 "prefix": "-moz-",
-                "version_added": "4"
+                "version_added": "4",
+                "version_removed": "120"
```

Comment:
Are we still doing `version_removed`? I seem to remember we omit the historic information.

Reply:
We do. It becomes historic once Firefox 120 was released for two years. Then we consider it irrelevant and safe to remove the information entirely.

Reply:
> once Firefox 120 was released for two years

Ah yes, thanks for the reminder 👍🏻